code/modules/urist/gamemodes/scom/scom.dm

```diff
@@ -0,0 +1,70 @@
+//if things aren't commented, i'm sorry. if you want help porting this to whatever codebase you're using, just ask me. i don't bite. scrdest, if you're fixing anything i fucked up, i'll give you a hand.
+//but to be frank, i assume i'm going to be the only person touching this stuff (love you scrdest) so that's why. - Glloyd (who else would it be?)
+
+/obj/item/var/scommoney = null //only used for science now.
+
+var/missiondiff = 1
+var/scommapsloaded = 0
+
+var/sploded = 0
+
+var/list/scomspawn1 = list()
+var/list/scomspawn2 = list()
+var/list/scomspawn3 = list()
+
+/datum/game_mode/scom //the joke is that 90% of this stuff is handled by other procs
+	name = "scom"
+	config_tag = "scom"
+	required_players = 0
+	required_players_secret = 18
+	votable = 0
+
+/datum/game_mode/announce() //guys, are my comments informative yet?
```

Comment:
Shouldn't it be /game_mode/scom/announce? Otherwise you're overriding announce()